TouchPortalAPI/client.py

```diff
@@ -420,10 +420,12 @@ def choiceUpdate(self, choiceId:str, values:list):
         This updates the list of choices in a previously-declared TP State with id `stateId`.
         See TP API reference for details on updating list values.
         """
+        
         if choiceId:
             if isinstance(values, list):
-                self.send({"type": "choiceUpdate", "id": choiceId, "value": values})
-                self.choiceUpdateList[choiceId] = values
+                if self.choiceUpdateList.get(choiceId) != values:
```

Comment:
What does this even do, actually?  Member-wise array comparison?  No idea offhand how JS compares arrays.  Most languages would just compare that the values _point to_ the same array instance, not actually compare the values of each array member against each other.